cluster-operator/src/main/java/io/strimzi/operator/cluster/model/EntityTopicOperator.java

```diff
@@ -77,7 +77,7 @@ protected EntityTopicOperator(String namespace, String cluster, Labels labels) {
         this.watchedNamespace = namespace;
         this.reconciliationIntervalMs = EntityTopicOperatorSpec.DEFAULT_FULL_RECONCILIATION_INTERVAL_SECONDS * 1_000;
         this.zookeeperSessionTimeoutMs = EntityTopicOperatorSpec.DEFAULT_ZOOKEEPER_SESSION_TIMEOUT_SECONDS * 1_000;
-        this.topicConfigMapLabels = defaultTopicConfigMapLabels(cluster);
+        this.topicConfigMapLabels = ModelUtils.defaultResourceLabels(cluster);
```

Comment:
Is it better to change the variable name as they aren't configmap anymore. You used just resourceLabels in the UO.

Reply:
Ok, should be fixed now.